### ISSUE TITLE
MP4Upload (masterani.me) not actually downloading anything

### ISSUE THREAD

**Issue by freddiaN** [vn-ki/anime-downloader#71]
**Describe the bug**

`anime-downloader` does find all the information for the series, but doesn't actually download anything... or rather try and download something but just stop it without any error

**To reproduce**
Download a series that is hosted on mp4upload (for this example: https://www.masterani.me/anime/info/690-ookami-to-koushinryou)
```
anime dl "https://www.masterani.me/anime/info/690-ookami-to-koushinryou"
```

<details>
<summary>LOG</summary>

```
anime dl "https://www.masterani.me/anime/info/690-ookami-to-koushinryou" -ll DEBUG
INFO root: anime-downloader 3.5.0
DEBUG root: Platform: Linux-4.18.1-arch1-1-ARCH-x86_64-with-arch-Arch-Linux
DEBUG root: Python 3.7.0
INFO root: Extracting episode info from page
INFO root: Found anime: Ookami to Koushinryou
INFO root: Downloading to /home/jonas
DEBUG root: Extracting stream info of id: https://www.masterani.me/anime/watch/690-ookami-to-koushinryou/1
DEBUG root: [cfscrape] Calling https://www.masterani.me/anime/watch/690-ookami-to-koushinryou/1
DEBUG root: Found sources [{'id': 175738, 'host_id': 1, 'embed_id': '2pyyk7su6u5y', 'quality': 1080, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107376, 'host_id': 1, 'embed_id': 'k9g66w4rdfzp', 'quality': 720, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107380, 'host_id': 1, 'embed_id': 'dz2jeya02ace', 'quality': 720, 'type': 2, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107377, 'host_id': 1, 'embed_id': 'w187atdwdc94', 'quality': 480, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107381, 'host_id': 1, 'embed_id': '5k09ytil3a1e', 'quality': 480, 'type': 2, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}]. filtering...
DEBUG root: [('mp4upload', 'https://mp4upload.com/embed-2pyyk7su6u5y.html')]
DEBUG root: Sources :
INFO root: Downloading Ookami to Koushinryou-1
INFO root: ./Ookami_to_Koushinryou/Ookami_to_Koushinryou_1.mp4




```

</details>

(I couldn't copy more sadly... but it basicly just printed an empty line after this and then  started to download the next episode (and also fail)
also vn-ki told me to @ngomile because he apperantly wrote the mp4upload integration =]

**Comment by ngomile** [comment]
The regular expression is flawed, sorry about that but it's an easy fix. Just have to add a +? in the (www\d) so that it becomes (www\d+?). And the extracting of title from mp4upload is a bit off as well. I'm on it.

**Comment by vn-ki** [comment]
Closed in #72

**Comment by freddiaN** [comment]
So I just installed the latest version from git and I get this error now... this may be an error on my end, and if it is, it would be nice if someone could tell me how to fix this:

```

➜ anime dl "https://www.masterani.me/anime/info/690-ookami-to-koushinryou" -ll DEBUG
INFO root: anime-downloader 3.5.0
DEBUG root: Platform: Linux-4.18.1-arch1-1-ARCH-x86_64-with-arch-Arch-Linux
DEBUG root: Python 3.7.0
INFO root: Extracting episode info from page
INFO root: Found anime: Ookami to Koushinryou
INFO root: Downloading to /home/jonas
DEBUG root: Extracting stream info of id: https://www.masterani.me/anime/watch/690-ookami-to-koushinryou/1
DEBUG root: [cfscrape] Calling https://www.masterani.me/anime/watch/690-ookami-to-koushinryou/1
DEBUG root: Found sources [{'id': 175738, 'host_id': 1, 'embed_id': '2pyyk7su6u5y', 'quality': 1080, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107376, 'host_id': 1, 'embed_id': 'k9g66w4rdfzp', 'quality': 720, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107380, 'host_id': 1, 'embed_id': 'dz2jeya02ace', 'quality': 720, 'type': 2, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107377, 'host_id': 1, 'embed_id': 'w187atdwdc94', 'quality': 480, 'type': 1, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}, {'id': 107381, 'host_id': 1, 'embed_id': '5k09ytil3a1e', 'quality': 480, 'type': 2, 'host': {'id': 1, 'name': 'MP4Upload', 'embed_prefix': 'https://mp4upload.com/embed-', 'embed_suffix': '.html'}}]. filtering...
DEBUG root: [('mp4upload', 'https://mp4upload.com/embed-2pyyk7su6u5y.html')]
DEBUG root: Sources :
DEBUG root: Domain: www14, Video ID: qwx47bctz3b4quuofsuuqiszlbxs4olzornvijhzrj62lprvdkq3srwp, Protocol: 282
DEBUG root: Title is Spice_and_Wolf_-_01_-_1080p
DEBUG root: Stream URL: https://www14.mp4upload.com:282/d/qwx47bctz3b4quuofsuuqiszlbxs4olzornvijhzrj62lprvdkq3srwp/Spice_and_Wolf_-_01_-_1080p.mp4
INFO root: Downloading Ookami to Koushinryou-1
INFO root: ./Ookami_to_Koushinryou/Ookami_to_Koushinryou_1.mp4
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/urllib3/contrib/pyopenssl.py", line 444, in wrap_socket
    cnx.do_handshake()
  File "/usr/lib/python3.7/site-packages/OpenSSL/SSL.py", line 1907, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "/usr/lib/python3.7/site-packages/OpenSSL/SSL.py", line 1639, in _raise_ssl_error
    _raise_current_error()
  File "/usr/lib/python3.7/site-packages/OpenSSL/_util.py", line 54, in exception_from_error_queue
    raise exception_type(errors)
OpenSSL.SSL.Error: [('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 343, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 849, in _validate_conn
    conn.connect()
  File "/usr/lib/python3.7/site-packages/urllib3/connection.py", line 356, in connect
    ssl_context=context)
  File "/usr/lib/python3.7/site-packages/urllib3/util/ssl_.py", line 359, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/lib/python3.7/site-packages/urllib3/contrib/pyopenssl.py", line 450, in wrap_socket
    raise ssl.SSLError('bad handshake: %r' % e)
ssl.SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])",)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/requests/adapters.py", line 445, in send
    timeout=timeout
  File "/usr/lib/python3.7/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/lib/python3.7/site-packages/urllib3/util/retry.py", line 398, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www14.mp4upload.com', port=282): Max retries exceeded with url: /d/qwx47bctz3b4quuofsuuqiszlbxs4olzornvijhzrj62lprvdkq3srwp/Spice_and_Wolf_-_01_-_1080p.mp4 (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/anime", line 11, in <module>
    load_entry_point('anime-downloader==3.5.0', 'console_scripts', 'anime')()
  File "/usr/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/anime_downloader/cli.py", line 146, in
dl
    range_size=chunk_size)
  File "/usr/lib/python3.7/site-packages/anime_downloader/sites/anime.py", line
183, in download
    path, force, range_size=range_size)
  File "/usr/lib/python3.7/site-packages/anime_downloader/downloader/base_downloader.py", line 23, in __init__
    r = requests.get(self.url, stream=True)
  File "/usr/lib/python3.7/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python3.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3.7/site-packages/requests/sessions.py", line 512, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.7/site-packages/requests/sessions.py", line 622, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.7/site-packages/requests/adapters.py", line 511, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www14.mp4upload.com', port=282): Max retries exceeded with url: /d/qwx47bctz3b4quuofsuuqiszlbxs4olzornvijhzrj62lprvdkq3srwp/Spice_and_Wolf_-_01_-_1080p.mp4 (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])")))

```
